packages/metrics/src/features/audit/handler.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import * as Hapi from '@hapi/hapi'
+import { generateAuditPoint } from '@metrics/features/registration/pointGenerator'
+import { query, writePoints } from '@metrics/influxdb/client'
+import { internal } from '@hapi/boom'
+import { IUserAuditBody } from '@metrics/features/registration'
+import { PRACTITIONER_ID } from '@metrics/features/getTimeLogged/constants'
+
+export async function newAuditHandler(
+  request: Hapi.Request,
+  h: Hapi.ResponseToolkit
+) {
+  const points = []
+  try {
+    const remoteAddress =
+      request.headers['x-real-ip'] || request.info.remoteAddress
+    const userAgent = request.headers['user-agent']
```

Comment:
I'm thinking we could do the same for `user-agent` as we do with the IP address. So

```ts
const userAgent = request.headers['x-real-user-agent'] || request.headers['user-agent']
```